### ISSUE TITLE
When ''capture region'' is used file explorer starts

### ISSUE THREAD

**Issue by w3rty88** [ShareX/ShareX#4635]
So ... I 've been using sharex a couple of years on my windows 10 laptop - desktop on home and work. never had an issue. BUT from today something strange is happening. When is take a ''whole screenshot'' all good but anytime i choose a specific region to snap ''file explorer'' starts. And if I make 10 screenshots file explorer starts 10 times. I'll attach some files

ALL GOOD with ''Print scrn'' button.

1)
![4SLomXNycW](https://user-images.githubusercontent.com/61057325/75490655-3f112280-59bd-11ea-93fd-671c86da6042.png)

but when i took a specific region photo as shown down
2) 
![ShareX_EyNx2osw4O](https://user-images.githubusercontent.com/61057325/75490960-c8c0f000-59bd-11ea-8c6a-346ceaa92609.png)

Then this happens
![ShareX_1MOGX66GNc](https://user-images.githubusercontent.com/61057325/75490980-d37b8500-59bd-11ea-898a-52255ac65d47.png)

![ixtAWlDKuD](https://user-images.githubusercontent.com/61057325/75490982-d4141b80-59bd-11ea-978d-0291f9fb936b.png)

Very strange coz i didnt make any changes or something. Any ideas ? I tried to change the ''after capture'' => show files in explorer but now it opens the screenshot folder and the file explorer menu xD

edit: it seems that the problem is happening not only on sharex but also on one other program that is called gentool and is saving a replay from an online game (after every session it autosaves the replay of the game).

**Comment by Fuzzqt** [comment]
Most likely a windows update or windows system issue. I'd recommend going into your task manager processes (Ctrl + Shift + ESC will auto hotkey open this) > Right clicking Windows Explorer > End Task

File > Run new task

Type explorer.exe

restart your PC

If this doesn't solve your problem I recommend running CMD in Admin mode and executing 

sfc /scannow

There's most likely a corrupt file within your file explorer settings.

**Comment by w3rty88** [comment]
> Most likely a windows update or windows system issue. I'd recommend going into your task manager processes (Ctrl + Shift + ESC will auto hotkey open this) > Right clicking Windows Explorer > End Task
> 
> File > Run new task
> 
> Type explorer.exe
> 
> restart your PC
> 
> If this doesn't solve your problem I recommend running CMD in Admin mode and executing
> 
> sfc /scannow
> 
> There's most likely a corrupt file within your file explorer settings.

That's what came in my mind when i saw that it happens again on other occasions. Guess what ? I restarted explorer from the task manager and voila.... file explorer pops up again. 

I am doing now an sfc scan, ill check for any pending updates and see if the problem remains. 

Thnx a lot man :)

**Comment by w3rty88** [comment]
> There's most likely a corrupt file within your file explorer settings.

that's what i thought when i saw it happens on other apps also. i tried restarting explorer.exe from task manager and guess ? file explorer starts :P tried sfc scan and ''no integrity'' so i said f... it 
I went to the update manager install an update package just now (7 hours ago there was no update)
and for now the problem seems to be fixed. I made like 5-10 scnshots without explorer popping up. I'll see if it happens later during the day.

PS: As always shareX is great and it is still a great product without issues unlike windows :P 

Thnx for helping me man
